concepts/05 Widgets/Form/01 Getting Started with Form/50 Configure Item Labels.md

```diff
@@ -0,0 +1,187 @@
+Simple item [labels](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/label/) display the formatted field names of the [formData](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#formData) object. You can change the displayed text in the **label**.[text](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/label/#text) property. 
+
+The Form displays labels on the left side of editors. The text of all editors is aligned to the left. To change location of all labels, use the [labelLocation](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#labelLocation) property. Use **label**.[location](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/label/#location) property to relocate individual labels. You can change the horizontal alignment of individual labels in the [alignment](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/label/#alignment) property.
+
+Use the [showColonAfterLabel](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#showColonAfterLabel) property to display colons at the end of labels. Use the **label**.[showColon](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/label/#showColon) property for the same purpose for an individual label. Individual settings override common settings.
```

Comment:
Label properties are very straightforward, yet we spend three paragraphs describing them in detail. This section would be simpler to read if we said that individual labels are configured in the label object, and that some label props have counterparts that apply to all labels.